content/articles/creating-simple-visual-augmented-reality-application-in-android/index.md

```diff
@@ -0,0 +1,166 @@
+
+The next step is to download the plugin necessary for supporting Augmented Reality by importing 3D models and viewing them. To do this, go to settings (in Android Studio IDE), then to plugins and search for Google Sceneform Tools, install it, and restart your Android Studio.
+You should have some 3D models. You can download 3D objects from:
+[here](https://drive.google.com/folderview?id=1Ls6fzMiFTx8_uE7A6or_l6YnouJIFSsc). I would suggest you use a 3D object with over 75 points. 
+
+### Adding the 3D model to the project.
+
+After downloading the folders, you will copy the sampledata folder to the go-to Android Studio, right-click on the app section, select "open in," then select "Explorer." Paste the sampledata folder here.
+
+In the Java section, right-click and select the new folder "assets." Right-click on the new assets directory and select "open in Explorer" and paste the AR.src (from the one that you downloaded in the assets folder) file there. This is shown here.
+
+ ![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/assets.png)
+
+### Adding sceneform ux library
+
+We add the libraries to dependacies in build.grindle:(app)
+They are 
+```java
+    implementation 'com.google.ar.sceneform.ux:sceneform-ux:1.17.1'
+    implementation 'com.google.ar.sceneform:core:1.17.1'
+    implementation 'com.google.sceneform:1.17.1'
+```
+They are to be implemented as shown here 
+![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/dependencies.png)
+
+On the same build.gridle(:app) window add the highlighted codes.
+
+
+![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/dependenciesid.png)
+
+Add the highlighted classpath as shown here 
+
+![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/dependence.png)
+
+Lastly, add the plugin id in build.gridle(:app)
+
+ ![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/dependenciesid.png)
+
+
+
+### Adding permissions for camera
+
+On the AndroidManifest.xml page, add the following codes;
+
+```java    
+    <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-feature android:name="android.hardware.camera.ar" android:required="true"/>
+```
+
+They should be as shown here
+ ![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/permission.png)
+
+### Application design
+
+We now need to configure the layout of the application. On the activity_main.xml window under layout folder modify existing codes to these shown below.
+
+![alt text](/engineering-education/creating-simple-visual-augmented-reality-application-in-android/mainactivity.png)
+
+Note that we change the layout from the dafault layout to relative layout.
+
+### Java codes
+These codes are written in the MainActivity.Java window. These codes are:
```

Comment:
Explain what you are trying to implement using this code.
Explain each method used also.